jsk_pcl_ros_utils/src/pointcloud_to_pcd_nodelet.cpp

```diff
@@ -112,16 +112,16 @@ namespace jsk_pcl_ros_utils
     {
       if(compressed_)
       {
-        ROS_INFO_STREAM ("Saving as binary compressed PCD");
+        NODELET_INFO("Saving as binary compressed PCD") ;
       }
       else
       {
-        ROS_INFO_STREAM ("Saving as binary PCD");
+        NODELET_INFO("Saving as binary PCD");
       }
     }
     else
     {
-      ROS_INFO_STREAM ("Saving as binary PCD");
+      NODELET_INFO("Saving as binary PCD");
```

Comment:
I mean
```suggestion
      NODELET_INFO("Saving as ASCII PCD");
```

Reply:
i see

Reply:
Sorry to ask you again, but it looks like the commit is not reflected after all ...?

Reply:
sorry, maybe i'm a git beginner... please check

Reply:
LGTM!